CMakeLists.txt

```
@@ -296,8 +291,8 @@ enable_testing()
 add_test(run_no_command_line_arguments ${PROJECT_NAME})
 
 if(NOT ARP_DEVICE AND NOT MINGW AND NOT SKIP_DEMO)
-    add_test(T1-PLCnext-Demo lua5.1
-                -e "package.cpath = package.cpath..';${CMAKE_BINARY_DIR}/?.dll;${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/?.dll'"
+    add_test(T1-PLCnext-Demo ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/lua
```

```suggestion
    add_test(T1-PLCnext-Demo ${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/lua
```
add_test(T1-PLCnext-Demo lua5.1
                -e "package.cpath = package.cpath..';${CMAKE_BINARY_DIR}/?.dll;${CMAKE_BINARY_DIR}/${CMAKE_BUILD_TYPE}/?.dll'"